dataland-frontend/src/components/forms/CreateEUTaxonomyForFinancials.vue

```diff
@@ -148,7 +148,7 @@
                         </div>
                         <!-- Integrated report is on a group level -->
                         <div class="form-field">
-                          <YesNoComponent
```

Comment:
We should either migrate to the new v-for approach, or have a ticket to do so.

Reply:
I created stories for creating data model files and using them like in the LkSG framework for the other 3 frameworks.

Reply:
ok